src/init.cpp

```diff
@@ -634,6 +634,8 @@ void SetupServerArgs(ArgsManager& argsman)
                              "is of this size or less (default: %u)",
                              MAX_OP_RETURN_RELAY),
                    ArgsManager::ALLOW_ANY, OptionsCategory::NODE_RELAY);
+    argsman.AddArg("-relayinscription", strprintf("Relay and mine inscription transactions and remove the fee reduction for them (default: %u)", DEFAULT_RELAY_INSCRIPTION), ArgsManager::ALLOW_ANY, OptionsCategory::NODE_RELAY);
```

Comment:
I don't think it makes much sense to have two options here; setting the size to 0 should be sufficient to say "I don't want any of this in my mempool". Having the default be `MAX_BLOCK_WEIGHT` serves to avoid the limit.

Reply:
I agree, there is now only one.